Bangazon/Areas/Identity/IdentityHostingStartup.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
You can eliminate this duplicate startup file that the generator creates.

Reply:
Done. Thanks